examples/units_and_coordinates/README.txt

```diff
@@ -0,0 +1,6 @@
+.. _units_and_coordinates_examples:
+
+.. _units_and_coordinates-examples-index:
+
+Units and Coordinates
```

Comment:
I think potentially the `AIA_limb_STEREO.py` file might be more coordinates than plotting? (I leave it up to you)

Reply:
Okay yes I'll move this to coordinates.